marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/FlowMvcTest.java

```diff
@@ -25,12 +25,7 @@
     @Test
     void runIngestionStep() throws Exception {
         installReferenceModelProject();
-        if(isVersionCompatibleWith520Roles()) {
-            loginAsTestUserWithRoles("hub-central-step-runner");
-        }
-        else {
-            loginAsTestUserWithRoles("hub-central-step-runner", "flow-operator-role");
-        }
+        loginAsTestUserWithRoles("hub-central-step-runner");
```

Comment:
Is this change intentional. Srinath fixed it so test will pass in 9.0-12.

Reply:
Yes. I adjusted hub-central-step-runner to work with both 9.0-12 and 10.0-*

Reply:
Thanks Ryan. I deleted the other comment after seeing the existing flow was asserted and it seems to have deleted your comment as well. Sorry about that.